usr.bin/ncal/ncal.c

```diff
@@ -911,10 +919,13 @@ mkmonthb(int y, int m, int jd_flag, struct monthlines *mlines)
 	}
 
 	/*
-	 * Set firsts to the day number of sunday of the first week of
+	 * Set firsts to the day number of sunday or monday of the first week of
 	 * this month. (This might be in the last month)
 	 */
-	firsts = first - (weekday(first)+1) % 7;
+	if (flag_monday)
+		firsts = first - weekday(first);
+	else
+		firsts = first - (weekday(first)+1) % 7;
```

Comment:
Style nit: please add spaces around + here. Style has evolved since this was written.

Reply:
I also renamed `firsts` (first Sunday) to `firstsm` (first Sunday and Monday) in this function for little more compliance.